dispatcher/factories.py

```diff
@@ -24,12 +24,11 @@
 def process_manager_from_settings(settings: LazySettings = global_settings):
     cls_name = settings.service.get('process_manager_cls', 'ForkServerManager')
     process_manager_cls = getattr(process, cls_name)
-    return process_manager_cls()
+    return process_manager_cls(settings=settings)
```

Comment:
This will conflict with another fix, but these are both additive changes, on this line. So add both things.